.pre-commit-config.yaml

```diff
@@ -28,5 +28,5 @@ repos:
     rev: "v2.2.2"
     hooks:
       - id: codespell
-        args: ["-I", "codespell.txt"]
+        args: ["-I", ".github/codespell.txt"]
```

Comment:
I think it was correct before. It's a file tracked at the repo root

Reply:
As mentioned in the PR description, this PR aims to move the `codespell.txt` inside the `.github/` dir with other configuration files such as `config.yml`, `dependabot.yml` and `stale.yml`. I think it makes sense for all the configuration files to reside at the same place (in our case the `.github/` dir).

Reply:
ah, got it. Sorry, didn't read the description. However, the pre-commit file is synced between most of the 30 repos of this org (see the note on top of the file). So this would have to be done for all repos, and I don't think it's worth the work.

Reply:
> ah, got it. Sorry, didn't read the description. However, the pre-commit file is synced between most of the 30 repos of this org (see the note on top of the file). So this would have to be done for all repos, and I don't think it's worth the work.

Oh, got it. I'll revert the changes, maybe this is worth considering for the "blueprint maintenance repository". It's a minimal change which has no functional effect on the workings of the projects.

Reply:
well, also I'm not sure if I completely agree with the motivation. The configuration doesn't have much to do with `.github`  (in the same way that many other config files are typically at the root of the repo). So I would actually be surprised to find config files from other tools that are not github actions there